apps/toffee/dev/base/kustomization.yaml

```diff
@@ -11,5 +11,6 @@ resources:
 namespace: toffee
 patches:
```

Comment:
On this file, can we also add the resources file related to config?
https://github.com/hmcts/sds-flux-config/blob/master/apps/azureserviceoperator-system/resources/flexibleserver-postgres-config.yaml

Reply:
yep can do :)